docs/user-guide.md

```diff
@@ -205,9 +205,12 @@ The currently supported options are:
 | kompose.service.healthcheck.liveness.http_get_path  | kubernetes liveness httpGet path                                                     |
 | kompose.service.healthcheck.liveness.http_get_port  | kubernetes liveness httpGet port                                                     |
 | kompose.service.healthcheck.liveness.tcp_port       | kubernetes liveness tcpSocket port                                                   |
-| kompose.service.external-traffic-policy       | 'cluster', 'local', ''                                                   |                                                |
-| kompose.security-context.fsgroup       |  kubernetes pod security group fsgroup                                                  |                                                |
-| kompose.volume.sub-path                        | kubernetes volume mount subpath                                                   |                                                |
+| kompose.service.external-traffic-policy             | 'cluster', 'local', ''                                                               |
+| kompose.security-context.fsgroup                    | kubernetes pod security group fsgroup                                                |
+| kompose.volume.sub-path                             | kubernetes volume mount subpath                                                      |
+| kompose.cronjob.schedule                            | kubernetes cronjob schedule                                                          |
```

Comment:
sorry to be a pain, can you give "example" inputs for the cronjobs, like:
```
      kompose.cronjob.schedule: "* * * * *"
      kompose.cronjob.concurrency_policy: "Forbid"
      kompose.cronjob.backoff_limit: "0"
```

Just put in brackets what they would possibly be (ex. "Forbig / Allow / Never"), etc.
    
Honestly we need to improve this table with a separate column with examples... But that'll be in a later PR